src/main/java/de/blackforestsolutions/dravelopsdatamodel/TravelProvider.java

```diff
@@ -1,28 +1,67 @@
 package de.blackforestsolutions.dravelopsdatamodel;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import com.hazelcast.nio.serialization.Portable;
+import com.hazelcast.nio.serialization.PortableReader;
+import com.hazelcast.nio.serialization.PortableWriter;
 import lombok.Getter;
 import lombok.Setter;
 import lombok.experimental.Accessors;
-import lombok.extern.slf4j.Slf4j;
 
+import java.io.IOException;
+import java.io.Serializable;
 import java.net.URL;
 
+import static de.blackforestsolutions.dravelopsdatamodel.hazelcast.DravelOpsPortableFactory.DRAVEL_OPS_FACTORY_ID;
+import static de.blackforestsolutions.dravelopsdatamodel.hazelcast.DravelOpsPortableFactory.TRAVEL_PROVIDER_CLASS_ID;
+import static de.blackforestsolutions.dravelopsdatamodel.hazelcast.classdefinition.TravelProviderClassDefinition.*;
+
 @Getter
-@Slf4j
 @JsonDeserialize(builder = TravelProvider.TravelProviderBuilder.class)
-public final class TravelProvider {
+public final class TravelProvider implements Serializable, Portable {
+
+    private static final long serialVersionUID = -8728155372687060904L;
 
-    private final String name;
+    private String name;
 
-    private final URL url;
+    private URL url;
```

Comment:
Maybe we need this to implement Portable, but we arent immutable anymore in this case. How do we enforce to not use setter for this method?

Reply:
Done